draft-ietf-dnsop-svcb-https.md

```diff
@@ -1036,7 +1037,7 @@ Clients that restrict the destination port in "https" URIs
 same restriction to the "port" SvcParam.
 
 The presence of an HTTPS RR for an origin also indicates
-that clients should connect securely and use the "https" scheme, as
+that clients should always connect securely and use the "https" scheme, as
```

Comment:
"always" here is something I'm on the fence about.  I don't think it's safe to do a true HSTS-like pinning scheme based solely on an unauthenticated DNS record.  If DNSSEC is in play, then it looks better, but for regular HSTS we get the signal directly from the origin itself, and this is a more indirect mechanism that we may want to only use for the current batch of requests.

Reply:
"always" is wrong.  It's only as long as the HTTPS resource records are within their TTL.

Reply:
I meant "always" in the sense of "unconditionally", but evidently this is confusing.  Removed.